lib/osf-components/addon/components/draft-registration-card/template.hbs

```diff
@@ -1,59 +1,90 @@
 <div local-class="DraftRegistrationCard" data-test-draft-registration-card>
     <h4 local-class="DraftRegistratrionCard__title" data-test-draft-registration-card-title>
-        {{#if draftRegistration}}
-            {{draftRegistration.registrationSchema.name}}
+        {{#if this.draftRegistration}}
+            {{this.draftRegistration.registrationSchema.name}}
         {{else}}
-            {{#content-placeholders as |placeholder|}}
-                {{placeholder.text lines=1}}
-            {{/content-placeholders}}
+            <ContentPlaceholders as |placeholder|>
+                <placeholder.text @lines={{1}} />
```

Comment:
Does `@lines=1` not work?

Reply:
[According to the docs](https://github.com/emberjs/rfcs/blob/master/text/0311-angle-bracket-invocation.md#arguments), 

"If the argument value is a constant string, it can appear verbatim after the equal sign, i.e. `<FooBar @foo="some constant string" />`. Other values should be enclosed in curlies, i.e. `<FooBar @foo={{123}} @bar={{this.bar}} />`. Helpers can also be used, as in `<FooBar @foo={{capitalize this.bar}} />`."

Reply:
#til